### PR TITLE
Adjusting twitter card data for new twitter account

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Background
- We put in a request to twitter to have our twitter name changed from @HXHungryGeek to @TheHungryGeek
- The username was taken, but wasn't being used by the owner
- Twitter don't freely remove non-active users from twitter and change usernames but we were luckily very successful.
- I use twitter cards on The Hungry Geek to promote our blog posts and tweets - (https://cards-dev.twitter.com/validator)

### What does this PR do?
- This PR changes the twitter username.

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

### CODE REVIEW

**Comment by Joezo** [comment]
:+1:

**Comment by cameronviner** [comment]
I'll take the dev review

File: _includes/head.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
     <!-- Twitter Cards Meta Data - @HXHungryGeek -->
```
Comment (cameronviner) [inline _includes/head.html]: Should this be changed to  ?

**Comment by cameronviner** [comment]
Looks good +1

**Pull request merged**